<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>原神</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/yuanshen/yuanshen.png</normaloff>:/icon/yuanshen/yuanshen.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>简单ComboBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnInitItems_2">
          <property name="text">
           <string>初始化列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>清空列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="chkEditable">
          <property name="text">
           <string>可编辑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QComboBox" name="combItems">
          <item>
           <property name="text">
            <string>温迪</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon/yuanshen/1.png</normaloff>:/icon/yuanshen/1.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>钟离</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon/yuanshen/2.png</normaloff>:/icon/yuanshen/2.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>荧</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon/yuanshen/3.png</normaloff>:/icon/yuanshen/3.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>派蒙</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon/yuanshen/5.png</normaloff>:/icon/yuanshen/5.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>刻晴</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon/yuanshen/4.png</normaloff>:/icon/yuanshen/4.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>有用户数据ComboBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnInitCities">
          <property name="text">
           <string>初始化城市+区号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboCities">
          <property name="placeholderText">
           <string>请选择一个人物</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>显示信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnclear">
        <property name="text">
         <string>清空文本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
